index.bs

```diff
@@ -727,6 +741,7 @@ authorizing an authenticator with which to complete the operation.
         USVString                            rpId;
         sequence<ScopedCredentialDescriptor> excludeList = [];
         Attachment                           attachment;
+        KeyStorage                           keystorage; 
```

Comment:
I have many comments on this topic, see my review of #378. Let's leave it out of this PR.